packages/core/src/codecatalyst/devEnv.ts

```diff
@@ -246,84 +241,39 @@ export class InactivityMessage implements vscode.Disposable {
         userIsActive: () => void,
-        willRefreshOnStaleTimestamp: () => Promise<boolean>,
-        oneMin: number
+        willRefreshOnStaleTimestamp: () => Promise<boolean>
     ) {
         getLogger().debug(
             'InactivityMessage.show: minutesUserWasInactive=%d minutesUntilShutdown=%d',
             minutesUserWasInactive,
             minutesUntilShutdown
         )
 
-        // Hide any current message.
-        this.progressTimeout.dispose()
-
         if (await willRefreshOnStaleTimestamp()) {
             return
         }
 
-        if (minutesUntilShutdown <= 1) {
-            const imHere = "I'm here!"
-            return vscode.window
-                .showWarningMessage(
-                    `Your CodeCatalyst Dev Environment has been inactive for ${minutesUserWasInactive} minutes, and will stop soon.`,
-                    { modal: true },
-                    imHere
-                )
-                .then(res => {
-                    if (res === imHere) {
-                        userIsActive()
-                    }
-                })
-        }
-
-        // Show a new message every minute
-        this.progressTimeout = new Timeout(oneMin)
-        this.progressTimeout.token.onCancellationRequested(c => {
-            getLogger().debug('InactivityMessage.show: CancelEvent.agent=%s', c.agent)
-            if (c.agent === 'user') {
-                // User clicked the 'Cancel' button, indicate they are active.
-                userIsActive()
-            } else {
-                // The message timed out, show the updated message.
-                void this.show(
-                    devEnvActivity,
-                    minutesUserWasInactive + 1,
-                    minutesUntilShutdown - 1,
-                    userIsActive,
-                    willRefreshOnStaleTimestamp,
-                    oneMin
-                )
-            }
-        })
-        if (isCloud9()) {
-            // C9 does not support message with progress, so just show a warning message.
-            return vscode.window
-                .showWarningMessage(this.getMessage(minutesUserWasInactive, minutesUntilShutdown), 'Cancel')
-                .then(() => {
-                    this.progressTimeout.cancel()
-                })
-        } else {
-            // Show cancellable "progress" message.
-            return void showMessageWithCancel(
-                this.getMessage(minutesUserWasInactive, minutesUntilShutdown),
-                this.progressTimeout
+        const imHere = "I'm here!"
+        return vscode.window
+            .showWarningMessage(
+                `Your CodeCatalyst Dev Environment has been inactive for ${minutesUserWasInactive} minutes, and will stop soon.`,
+                { modal: true },
+                imHere
             )
-        }
+            .then(res => {
+                if (res === imHere) {
+                    userIsActive()
+                }
+            })
```

Comment:
what if the user X'es out the notification? That means they are active. Other than closing VSC, not sure if there are other ways that that a notification can enter this block without being explicitly interacted with.
```suggestion
            .then(userIsActive)
```

Reply:
> what if the user X'es out the notification? That means they are active.

Yep. Presumably if they hit Cancel or X, they are intentionally choosing not to click `I'm Here` because they don't want to trigger activity.